baremetal_network_provisioning/ml2/hp_network_provisioning_driver.py

```diff
@@ -34,7 +34,7 @@
     cfg.StrOpt('base_url',
                help=_("Base HTTP URL of  SDN controller")),
     cfg.StrOpt('auth_token',
-               default='AuroraSdnToken37',
+               default='password',
```

Comment:
Better to have the password read from an input file.

How do we expect the customer to change this in code?